src/site/stages/build/drupal/load-saved-flags.js

```diff
@@ -51,12 +55,14 @@ function useFlags(rawFlags) {
  * This is for use with the drupal-aws-cache script. The normal build
  * and preview build scripts do this in setUpFeatureFlags().
  */
-function loadFeatureFlags(cacheDirectory) {
+function loadFeatureFlags(cacheDirectory, buildType) {
   const featureFlagFile = path.join(cacheDirectory, 'feature-flags.json');
-  const rawFlags = fs.readJsonSync(featureFlagFile);
+  const rawFlags = fs.existsSync(featureFlagFile)
+    ? fs.readJsonSync(featureFlagFile)
+    : {};
```

Comment:
I'm wondering if this is necessary. The only time `loadFeatureFlags` is called is in the `drupal-aws-cache.js` script. That script is called to either load from or save the cache to the S3 bucket. If we're trying to load the cache, we'll need the "real" feature flags in order to construct the right query to get the cache key. If we're trying to save the cache to S3, we need the right flags to save the cache with the right key. :thinking: 

With that in mind, I think it's okay to fail the script if the feature flags file isn't found.

It's probably worth adding a comment to the function saying as much.